catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx

```diff
@@ -72,7 +72,7 @@ const ImageWrapper = styled(Space).attrs({
   ${props => props.theme.media('large')`
     flex: 0 1 auto;
     height: auto;
-    max-height: 350px;
+    max-height: 400px;
```

Comment:
The modal is still breaking but I think it's because `<IIIFImage>` within this component should have `layout="raw"`. It's not in the grid and doesn't need the specific styles that `"filled"` gives it.